infra/terraform/modules/data_access/organization_events/organization_events.py

```diff
@@ -0,0 +1,54 @@
+import json
+import os
+
+import boto3
+
+
+LAMBDA_ARNS = {
+    "create_user_role": os.environ["CREATE_ROLE_ARN"],
+    "delete_user_role": os.environ["DELETE_ROLE_ARN"],
+}
+
+
+def event_received(sns_event, context):
+    print("SNS event received = {}".format(json.dumps(sns_event)))
```

Comment:
It'd be better to use the `logging` module rather than `print`, so we can use log levels.